index.html

```diff
@@ -126,7 +127,35 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Laravel</a>
+                        <div class="cd-faq-content">
+                                <p>Eloquent (dostęp do bazy danych):</p>
+                                <ol>
+                                        <li>Dostępne są dwa query buildery: użycie <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Builder.php">pierwszego z nich</a> powoduje zwrócenie obiektów Active Record, natomiast <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Query/Builder.php">drugi</a> zwraca tablice.</li>
+                                        <li>Eloquent\Builder nie pozwala na filtrowanie wyników zapytania po wartościach z JOINowanych tabel i zwrócenie instancji modelu jednocześnie. <a href="https://www.facebook.com/groups/laravelpolska/permalink/1587069624864733/">Dyskusja</a>.</li>
+                                        <li>Wołając na obiekcie AR metodę <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L1460">save()</a> zwrócona wartość pozwoli nam ustalić czy operacja się powiodła. Używając metody <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L525">create()</a> powyższej informacji dostaczy nam <a href="https://github.com/laravel/framework/blob/v5.0.32/src/Illuminate/Database/Eloquent/Model.php#L175">publiczna właściwość klasy modelu</a>. W żadnym z powyższych wypadków nie zostanie automatycznie przerwana transakcja. <a href="https://www.facebook.com/groups/223889134481096/permalink/336420516561290/">Dyskusja</a>.</li>
+                                </ol>
+                                <p>Dostęp statyczny:</p>
+                                <ol>
+                                		<li>W Laravelu dostępny jest moduł Dependency Injection oparty o Reflection i <a href="http://laravel.com/docs/5.1/container#resolving">mogący działać bez żadnej konfiguracji</a>. Znajdowały się w nim pewne <a href="https://github.com/laravel/framework/pull/4824">niedoskonałości</a> mogące skutecznie utrudnić debugowanie.</li>
```

Comment:
Nie piszę, że Taylor nie merdżuje PRów. Zwracam uwagę na to, że nawet te fragmenty kodu, które wydają się być dość sensowne mają zwyczaj być niedopracowane.